chajda/tsquery/augments.py

```diff
@@ -53,114 +58,225 @@ def augments_gensim(lang, word, config=Config(), n=5):
+# and the corresponding langauge as keys
 fasttext_models = {}
+
+
+def load_fasttext(lang):
+    '''
+    Downloads and loads a fasttext model for a particular language,
+    storing the fasttext model object in the fasttext_models dictionary.
+    Due to the size of fasttext models (>1gb), and the loading time,
+    we do not load all models on program startup.
+    Instead, this function is used to load them lazily as needed.
+    lang: language passed from augments_fasttext
+
+    NOTE:
+    The doctest below has been commented out due to requiring more memory than github actions allows for.
+    # >>> load_fasttext_model('ja') or fasttext_models['ja'] != None
+    # True
+    '''
+
+    working_dir = os.getcwd()
+    # suppress_stdout_stderr() is used for redirecting stdout and stderr when downloading and loading fasttext models.
+    # Without this, stdout and stderr from fasttext causes doctests to fail.
+    with suppress_stdout_stderr():
+
+        # fasttext downloads models to the current directory,
+        # but we want to store the model in chaja/tsquery/fasttext,
+        # so we change the working directory before downloading
+        try:
+            os.chdir(f"{fasttext_annoy_data_path}/fasttext")
+        except OSError:
+            os.mkdir(f"{fasttext_annoy_data_path}/fasttext")
+            os.chdir(f"{fasttext_annoy_data_path}/fasttext")
+
+        # download and load fasttext model
+        fasttext.util.download_model(lang, if_exists='ignore')
+        fasttext_models[lang] = fasttext.load_model(f"cc.{lang}.300.bin")
+
+        # fasttext downloads two different files for each model,
+        # however, we only need cc.{lang}.300,bin,
+        # so we remove unneeded .gz file if exists
+        if os.path.isfile(f"{fasttext_annoy_data_path}/fasttext/cc.{lang}.300.bin.gz"):
+            os.remove(f"{fasttext_annoy_data_path}/fasttext/cc.{lang}.300.bin.gz")
+
+        # change working directory back
+        os.chdir(working_dir)
+
+    # Executing fasttext's get_nearest_neighbor as well as accessing the words in the fasttext model both run
+    # significantly faster after they has been called once,
+    # so we call them once upon loading the model to ensure most efficient runtime for subsequent uses.
+    fasttext_models[lang].get_nearest_neighbors('google', k=5)
+    fasttext_list = []
+    for word in fasttext_models[lang].get_words(on_unicode_error='replace'):
+        fasttext_list.append(word)
+
+
+def destroy_fasttext(lang):
+    '''
+    removes fasttext model from disk and memory
+    '''
+
+    # remove fasttext download from disk
+    if os.path.isfile(f"{fasttext_annoy_data_path}/fasttext/cc.{lang}.300.bin"):
+        os.remove(f"{fasttext_annoy_data_path}/fasttext/cc.{lang}.300.bin")
+
+    # remove fasttext model from memory
+    try:
+        del fasttext_models[lang]
+    except KeyError:
+        pass
+
+
+#################################################################################
```

Comment:
You have 82 `#`s; there should be exactly 80; most formatting tools assume that there will be no more than 80 characters per line (you'll notice github causes it to bleed onto a second line); it's okay to exceed this number occasionally to improve readability of code/comments, but there's no reason to exceed it here